AlphaWallet.xcworkspace/xcshareddata/xcschemes/AlphaWallet.xcscheme

```diff
@@ -96,6 +96,11 @@
             value = ""
             isEnabled = "YES">
          </AdditionalOption>
+         <AdditionalOption
```

Comment:
Reresolves issue with growing app size over using the app. Disable loggin all mallocs to file.